mindmeld/cli.py

```diff
@@ -85,6 +91,112 @@ def _app_cli(ctx):
         ctx.obj = {}
 
 
+def _dvc_add_helper(filepath):
+    """
+    Returns True if successful, False otherwise along with helper message
+    :param path:
+    :return:
+    """
+    p = subprocess.Popen(
+        ["dvc", "add", filepath], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )
+    # Get DVC error message from standard error
+    _, error = p.communicate()
+    error_string = error.decode("utf-8")
+
+    if DVC_INIT_ERROR_MESSAGE in error_string:
+        return False, DVC_INIT_HELP
+    elif DVC_ADD_DOES_NOT_EXIST_MESSAGE in error_string:
+        return False, DVC_ADD_DOES_NOT_EXIST_HELP.format(dvc_add_path=filepath)
+    else:
+        return True, None
+
+
+@_app_cli.command("dvc", context_settings=CONTEXT_SETTINGS)
+@click.pass_context
+@click.option("--init", is_flag=True, required=False,
+              help="Instantiate DVC within a repository")
+@click.option("--save", is_flag=True, required=False,
+              help="Save built models using dvc")
+@click.option("--checkout_hash", required=False, help="Instantiate DVC within a repository")
+def dvc(ctx, init, save, checkout_hash):
+    app = ctx.obj.get("app")
+    app_path = app.app_path
+
+    if init:
+        p = subprocess.Popen(
+            ["dvc", "init"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+        )
+        _, error = p.communicate()
+        error_string = error.decode("utf-8")
+
+        if error_string:
+            logger.error("Error during initialization: %s", error_string)
+            return
+
+        # Set up a local remote
+        local_remote_path = os.path.join(app_path, "dvc_local_remote")
+        p = subprocess.Popen(
```

Comment:
I think there is some scope to abstracting out the process for:
1. issuing an OS command
2. Gathering the errors and output
3. Logging errors when needed

I see it happen quite a bit in this PR. Maybe we can refactor it into a function